public/docs/_examples/hierarchical-dependency-injection/ts/src/app/hero-tax-return.component.ts

```diff
@@ -15,10 +15,11 @@ import { HeroTaxReturnService } from './hero-tax-return.service';
 export class HeroTaxReturnComponent {
   message = '';
   @Output() close = new EventEmitter<void>();
-  @Input()
+
   get taxReturn(): HeroTaxReturn {
     return this.heroTaxReturnService.taxReturn;
   }
+  @Input()
```

Comment:
As a matter of style, we should encourage readers to associate `@Input` with setters. This way, documentation and refactoring tools, for example, will attribute the decorator to the proper class member.

Right, Angular's behavior isn't affected (though IMHO it might be incidental behavior that it happens to work when attached to the getter).

If you feel strongly about it, I can undo it. Let me know.